package.json

```diff
@@ -51,21 +48,21 @@
     "lint": "concurrently -r \"npm:lint:dev\" \"npm:lint:json\"",
     "npmcopy": "node build/tasks/npmcopy.js",
     "prepare": "husky install",
-    "pretest": "npm run qunit-fixture && npm run babel:tests && npm run npmcopy",
+    "pretest": "npm run npmcopy && npm run qunit-fixture && npm run babel:tests && npm run bundlers:tests",
```

Comment:
I'd like to avoid running any tests in the pretest script. It should only needs to run once. But, if it does turn out to be something we want to run on every build, it could instead be added to the build, similar to `dist()` and `compareSize()`.

Reply:
These are not tests, these are bundler runs for later testing as part of core unit tests.

Reply:
The main complication is bundlers… well, bundle, so this process does depend on the build being done beforehand.

Reply:
I think my suggestion still stands. It can be separate from the pretest script and only build files specific to the bundler tests when testing the bundle.

Reply:
The problem is without the bundler runs core unit tests now fail as of this PR. So it kind of is a pre-requisite to running unit tests.

I agree this setup is not ideal. I was thinking about creating a separate bundlers QUnit module just for these tests but I’d need to organize it somehow so that this module is only run when requested explicitly.

Another option would be trying to simulate a browser via jsdom, then this would be another set of Node-only tests.

I’ll think about it.

> only build files specific to the bundler tests when testing the bundle

These are `exports` tests, it’s the safest to build everything before running them.

Reply:
I migrated the tests to run on Node + JSDOM so this issue no longer exists.